Readme.MD

```diff
@@ -945,3 +945,79 @@ import { PricingCard } from 'react-native-elements'
 | pricingFont | System font (font weight 700) (iOS), Roboto-Bold (android) | string | specify pricing font family |
 | infoFont | System font bold (iOS), Roboto-Bold (android) | string | specify pricing information font family |
 | buttonFont | System font (iOS), Roboto (android) | string | specify button font family |
+
+## Grid Component
+The Grid component provides two types of layouts, Row and Column.
+
+#### Row
+
+![Row Component](http://i.imgur.com/1e8W6j6.png)
+
+```js
+import {Grid, Row} from 'react-native-elements';
+
+<Grid>
+  <Row></Row>
+  <Row></Row>
+</Grid>
+```
+
+#### Column
+
+![Column Component](http://i.imgur.com/MWIBLKk.png)
```

Comment:
The image looks really big. Consider using the `<img>` tag as that let's you control the size of the image. 
`<img src="example.png" width="850" height="186.72">`